qemu/tests/cfg/netkvm_in_use.cfg

```diff
@@ -0,0 +1,65 @@
+- netkvm_in_use:
+    type = driver_in_use
+    only Windows
+    only virtio_net
+    hostpassword = redhat
+    login_timeout = 800
+    start_vm = yes
+    kill_vm_on_error = yes
+    check_guest_bsod = yes
+    driver_name = "netkvm"
+    run_bgstress = netperf_stress
+    bg_stress_run_flag = netperf_run
+    netperf_link = netperf-2.6.0.tar.bz2
+    server_path = /var/tmp/
```

Comment:
The netperf cfg is for the host.
The guest windows part has been updated in the Windows.cfg.

Thanks.